pkg/monitor/cluster/clusterflagsandbanner.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	operatorFlagMetricsTopic  = "cluster.nonstandard.operator.featureflags"
+	supportBannerMetricsTopic = "cluster.nonstandard.contact.support.banner"
+)
+
+func (mon *Monitor) emitOperatorFlagsAndSupportBanner(ctx context.Context) error {
+	cs, err := mon.listAROClusters(ctx)
+	if err != nil {
+		return err
+	}
+
+	for _, cluster := range cs.Items {
+		if cluster.Spec.OperatorFlags != nil {
+			defualtFlags := api.DefaultOperatorFlags()
+			nonStandardOperatorFlags := make(map[string]string)
```

Comment:
nit: you can set the initialized length of this map to the length of the default flags